src/assets/scss/core/_custom.scss

```diff
@@ -741,9 +741,25 @@ table.color-variables {
       max-height: 450px;
     }
 
```

Comment:
The below changes are just for the Docs' display, and not part of the Design System itself.